core/src/main/java/de/muenchen/allg/itd51/wollmux/former/ComboboxMergeDescriptor.java

```diff
@@ -28,22 +28,22 @@
 import de.muenchen.allg.itd51.wollmux.former.model.IdModel;
 
 /**
- * Enthält Informationen über eine erfolgte verschmelzung mehrerer Checkboxen zu
- * einer einzigen Combobox.
+ * Contains information about a happened fusing together of multiple checkboxes to
+ * a single combobox.
```

Comment:
Hmm. How about: Information container for merging separate checkboxes into a single combobox ?